app/config/locale/translations/hu.json

```diff
@@ -3,30 +3,30 @@
     "settings.locale": "hu",
     "settings.direction": "ltr",
     "emails.sender": "%s Csapat",
-    "emails.verification.subject": "",
-    "emails.verification.hello": "",
-    "emails.verification.body": "",
-    "emails.verification.footer": "",
-    "emails.verification.thanks": "",
-    "emails.verification.signature": "",
-    "emails.magicSession.subject": "",
-    "emails.magicSession.hello": "",
-    "emails.magicSession.body": "",
-    "emails.magicSession.footer": "",
-    "emails.magicSession.thanks": "",
-    "emails.magicSession.signature": "",
-    "emails.recovery.subject": "",
-    "emails.recovery.hello": "",
-    "emails.recovery.body": "",
-    "emails.recovery.footer": "",
-    "emails.recovery.thanks": "",
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "Fiók Megerősítése",
+    "emails.verification.hello": "Szia {{name}}",
+    "emails.verification.body": "Kattints a linkre, hogy megerősítsd az email címedet.",
+    "emails.verification.footer": "Ha nem te kérted a címed megerősítését, akkor nyugodtan hagyd figyelmen kívül ezt az üzenetet.",
+    "emails.verification.thanks": "Köszönettel",
+    "emails.verification.signature": "a {{project}} csapat",
+    "emails.magicSession.subject": "Bejelentkezés",
+    "emails.magicSession.hello": "Szia,",
+    "emails.magicSession.body": "Kattints a linkre a bejelentkezéshez.",
+    "emails.magicSession.footer": "Ha nem te szerettél volna bejelentkezni ezzel az email címmel, akkor nyugodtan hagyd figyelmen kívül ezt az üzenetet.",
+    "emails.magicSession.thanks": "Köszönettel",
+    "emails.magicSession.signature": "a {{project}} csapat",
+    "emails.recovery.subject": "Jelszó Visszaállítása",
+    "emails.recovery.hello": "Hahó, {{name}}",
+    "emails.recovery.body": "Kattints a linkre a {{project}} jelszavad visszaállításához.",
+    "emails.recovery.footer": "Ha nem te kezdeményezted a jelszavad visszaállítását, akkor nyugodtan hagyd figyelmen kívül ezt az üzenetet.",
+    "emails.recovery.thanks": "Köszönettel",
+    "emails.recovery.signature": "a {{project}} csapat",
+    "emails.invitation.subject": "Meghívó a %s csapatba @ %s",
```

Comment:
is the @ character intentional here ?

Reply:
Does it have any special meaning in the language that I'm missing ?

Reply:
Heyo!

As I described in the original PR, 'at' has no trivial conjugation in Hungarian, which as a simple string would fit every context.
So I reverted to the internationally recognized symbol of @ which means 'at'

Reply:
In that case we need a bit more thinking around this. This would not be consistent with our other translations.

Reply:
could you maybe send me (or point me at) some examples for the whole sentences or some %s parameters so I could go on and try to formulate an universal string?

Reply:
could you maybe send me (or point me at) some examples for the whole sentences or some %s parameters so I could go on and try to formulate an universal string?

Reply:
![Screenshot from 2021-10-06 16-58-28](https://user-images.githubusercontent.com/20852629/136194216-517a30a7-39d6-4ba2-b28a-b43a9414943e.png)

Here's an example 

%s -> Arsenal 
%s -> Demo Project